cost/azure/unused_sql_databases/README.md

```diff
@@ -1,60 +1,40 @@
-## Azure Unused SQL Databases
- 
-### What it does
+# Azure Unused SQL Databases
+
+## What it does
 
 This Policy template checks for Azure SQL Databases that are unused by reviewing the DB connections and delete them after user approval.
 
-### Functional Details
+## Functional Details
 
 This policy gets a list of Azure SQL Databases and uses the DB Connection metric to check for successful connections over a 30-day period. If there are no successful DB Connections the policy will terminate the SQL databases after the user approval.
- 
-### Input Parameters
- 
+
+## Input Parameters
+
 This policy has the following input parameters required when launching the policy.
 
 - *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created
-- *Azure AD Tenant ID* - the Azure AD Tenant ID used for the Azure API Authentication
-- *Azure Subscription ID* - the Azure Subscription ID used for the Azure API Authentication
 - *Exclusion Tag Key* - Azure-native SQL Database tag key to ignore databases. Only supply the tag key. The policy assumes that the tag value is irrelevant.
 
-### Prerequisites
+## Prerequisites
 
-- Azure Service Principal (AKA Azure Active Directory Application) with the appropriate permissions to manage resources in the target subscription
-- The following RightScale Credentials
-  - `AZURE_APPLICATION_ID`
-  - `AZURE_APPLICATION_KEY`
-  
-### Installation
+This policy requires the Azure Resource Manager Credential. When applying the policy select the appropriate credentials from the list for your tenant. If such credential doesn't exist please contact your cloud admin to create the Credential.
```

Comment:
same as other readmes

Reply:
Earlier referred POL-151 for Prerequisites text, now changed it by referring to the policies which have been merged to master recently viz. Azure Rightsize SQL Databases and Azure Inefficient Instance Utilization using Log Analytics.